modules/core/05-port/types/ibc_legacy_module.go

```diff
@@ -453,13 +448,28 @@ func (*LegacyIBCModule) UnmarshalPacketData(ctx sdk.Context, portID, channelID s
 	return nil, nil
 }
 
+// reverseCallbacks returns a copy of the callbacks in reverse order.
+// the majority of handlers are called in reverse order, so this can be used
+// in those cases to prevent needing to iterate backwards over the callbacks.
+func (im *LegacyIBCModule) reverseCallbacks() []ClassicIBCModule {
```

Comment:
[nit] since this doesn't actually reverse the callbacks, but returns the "reversed", what about calling it `reversedCallbacks`?

Reply:
sgtm, I can't think of any better naming tbh